packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -520,6 +562,22 @@ impl PostgresDatabase {
         Ok(ids)
     }
 
+    async fn link_type_uris_to_version_ids<'p, I>(
```

Comment:
Note that this is slightly different to the other ones 🙃 

It doesn't take LinkTypeReferences because the link type is only 'referred' to in the key of an object. There is no "$ref"